src/planscape/plan/views.py

```diff
@@ -520,6 +521,27 @@ def get_scenario(request: HttpRequest) -> HttpResponse:
         return JsonResponse(ScenarioSerializer(scenario_exists).data, safe=False)
     except Exception as e:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
+    
+@csrf_exempt
+def list_scenarios_for_plan(request: HttpRequest) -> HttpResponse:
+    try:
+        assert isinstance(request.GET['plan_id'], str)
+        plan_id = request.GET.get('plan_id', "0")
+        plan_exists = Plan.objects.get(id=plan_id)
```

Comment:
Maybe this was already addressed in a previous pr:
1) Can this variable be named plan?
2) and is it possible to call get_user followed by get_plan_by_id?

Reply:
1. renamed
2. this method uses a different name for the id field, so leaving things as is for now.

Reply:
makes sense!
I have a preference for consistent names for url parameters (maybe get_plan_by_id should use "plan_id", too) for all entry points.
but that's a TODO for another day.